README.md

```diff
@@ -11,7 +11,9 @@ In `mix.exs`, add the ExMachina dependency:
 
 ```elixir
 def deps do
-  [{:ex_machina, "~> 0.4"}]
+  [
+    {:ex_machina, "~> 0.4", only: "test"}
```

Comment:
@henrik Good catch. We typically do that in the code itself, but in the README, I think it just needs to show you how to add it to your project in a concise way.